### PR TITLE
B lorient765

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
It's not online unfortunately

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient765.xml [hunk 1]
```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+          <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
+            </handNote>
+
+    </handDesc>
+    <additions>
+
+<list>
+
+<item xml:id="e1">
+<desc type="MixedNote">The name of the first owner has been erased, and that of <foreign xml:lang="gez">ክንፈ፡ ሚካኤል፡</foreign>substituted.</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient765.xml:123]: please encode the name with <persName> and role

File: LondonBritishLibrary/orient/BLorient765.xml [hunk 2]
```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+<msPart xml:id="p1">
+<msIdentifier></msIdentifier>
+<msContents>
+
+<msItem xml:id="p1_i1">
+<locus target="#3"/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient765.xml:87]: 3? the text of the second part start on fol. 3 as well...

File: LondonBritishLibrary/orient/BLorient765.xml [hunk 3]
```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+</msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+
+<msItem xml:id="p2_i1">
+<locus target="3r"/>
+<title type="complete" ref="LIT2169Pateri"/>
+</msItem>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient765.xml:142]: please, add textLang

File: LondonBritishLibrary/orient/BLorient765.xml [hunk 4]
```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus target="#155v"></locus>
+<locus target="#156"></locus>
+<locus target="#157"></locus>
+<desc type="MixedNote">Notes in Amharic from the reigns of
+  <persName ref="PRS3692egwalas">Gʷālu</persName>, <persName ref="PRS5637IyoasI">ʾIyoʾās</persName>, <persName ref="PRS5619IyasuIV">ʾIyāsu IV</persName> and <persName ref="PRS9430Tewodros">Theodore</persName>.</desc>
+<q xml:lang="am"/>
+</item>
+
+<item xml:id="e2">
+<desc type="MixedNote">The name of the first owner <foreign xml:lang="gez">አ<gap reason="lost"/>ተ፡ ክርስቶስ፡</foreign> <locus target="#38r"/> has been
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient765.xml:194]: no type, and the names of the owners should be encoded as <persName> with role. I think, you have already done the entries...

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 2 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient765.xml:87]: It is correct here, but see below

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 3 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient765.xml:142]: Wright does not provide with the beginning of the text, fol. 3r relates to another manuscript. Just take locus out, it is not obligatory.

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient765.xml:123]: Done.

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient765.xml:142]: Deleted locus.

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient765.xml:87]: So I'm leaving it as it is.

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 4 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient765.xml:194]: Done.

File: LondonBritishLibrary/orient/BLorient765.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient765.xml:141]: Added textLang, below as well.

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**